File: week-3/day-1/primrow.py
def isprim(num):
    has_divisor = 0
    i = 2
    while i < num and num != has_divisor:
        if num % i == 0:
            has_divisor = 1
        i += 1
    if has_divisor or num == 0 or num == 1:
        return False
    else:
        return True

n = 0
while n < 100:
    if isprim(n):
        print(n)
    n += 1

File: holidaJS/primrow.js
function isPrim(num) {
    var hasDivisor = 0;
    var i = 2;
    while (i < num) {
        if (num % i ==0) {
            hasDivisor = 1;
        }
        i++
    }

    if(hasDivisor ==1 || num == 0 || num == 1) {
        return false;
    }else {
        return true;
    }
}


function primerow(num) {
    var i=0;
    while (i<num) {
        if (isPrim(i) == true) {
            console.log(i + " is prime");
            i++
        } else if (isPrim(i)== false) {
            console.log(i + " is not prime");
            i++
        }
    }
}

primerow(50)
